src/services/swap_service.ts

```diff
@@ -10,6 +10,7 @@ import {
     SwapQuoter,
     SwapQuoterOpts,
```

Comment:
I guess the question still remains: can we skip the `_estimateGasOrThrowRevertErrorAsync` if we are doing a RFQT firm quote?  

I imagine we'd like to still return a firm quote to a taker even if they do not have allowances or a sufficient asset balance.  What do you think?

Reply:
AFAIK RFQT for a dex-aggregator contract will still be broken until we resolve this.

Consider a dex.ag smart contract like 1inch.  The takerAddress would be the 1inch contract.  The `_estimateGasOrThrowRevertErrorAsync` call would fail because the `from` address would be the 1inch contract which does not hold a balance of the taker asset.

Suggestion: for now, disable eth gas estimation for RFQT requests

Reply:
As discussed with Steve and Jacob elsewhere, we decided to introduce a new query parameter, `skipValidation`.  An integrator using a smart contract as a taker can use this parameter to suppress validation of the transaction.  All other integrators will get the the default, which is to validate.  Implemented in https://github.com/0xProject/0x-api/pull/162/commits/ea8c8e94752e2b07c7d2f60b21ac2acc8e4f01de